olm-catalog/serverless-operator/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM scratch
+
+COPY manifests /manifests
+COPY metadata/annotations.yaml /metadata/annotations.yaml
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=serverless-operator
+LABEL operators.operatorframework.io.bundle.channel.default.v1="4.5"
```

Comment:
Previously, we had the default channel `4.3` . Why is it changing now? We're still targeting 4.5 ?

Reply:
OK. This is fixed. Thanks